# PLEASE NOTE THIS FILE IS NOT USED AUTOMATICALLY BY THE ODOMETRY'S LAUNCH FILE
# it's meant to just tell you what are the defaults for all the parameters
# to use this config with the launch file, you will still have to pass it with
# `config_file:=<full_or_relative_path_to>/ros_default.yaml`
# #############################################################################
# base_frame: base
# imu_topic: /imu
# imu_frame: <leave empty if tf is well defined>
# lidar_topic: /lidar
# lidar_frame: <leave empty if tf is well defined>
# extrinsic_lidar2base_quat_xyzw_xyz: [qxyzw_xyz as a list, unnecessary if tf is well defined]
# extrinsic_imu2base_quat_xyzw_xyz: [qxyzw_xyz as a list, unnecessary if tf is well defined]
odom_topic: /rko_lio/odometry
odom_frame: odom
invert_odom_tf: false
publish_local_map: true
map_topic: /rko_lio/local_map
publish_deskewed_scan: true
# lio config
deskew: true
voxel_size: 1.0
max_points_per_voxel: 20
max_range: 100.0
min_range: 1.0
max_correspondance_distance: 0.5
convergence_criterion: 0.00001
max_num_threads: 0
initialization_phase: false
double_downsample: true
max_expected_jerk: 3.0
min_beta: 200.0
